docs/master/api-reference/directives.md

```diff
@@ -2156,6 +2178,64 @@ type PostEdge {
 }
 ```
 
+### Simple Pagination
+
+The default `PAGINATOR` uses the query builder method `paginate` which will
+fire an additional query every time to count the total number of items. This
+is usually very helpful to generate pagination elements in the frontend.
+
+If you wish to use the `simplePaginate` method, set the `type` to `SIMPLE`.
+
+> Please note that the `SIMPLE` paginator does not have the attributes
+> `hasMorePages`, `lastPage` and `total`.
+>
+> If you need those fields, you should use the default `PAGINATOR`.
+
+```graphql
+type Query {
+  posts: [Post!]! @paginate(type: SIMPLE)
+}
+```
+
+The schema definition is automatically transformed to this:
+
+```graphql
+type Query {
+  posts(
+    "Limits number of fetched elements."
+    first: Int!
+
+    "The offset from which elements are returned."
+    page: Int
+  ): PostSimplePaginator
+}
+
+"A paginated list of Post items."
+type PostSimplePaginator {
+  "A list of Post items."
+  data: [Post!]!
+
+  "Pagination information about the list of items."
+  paginatorInfo: SimplePaginatorInfo!
```

Comment:
Let's include the definition for `SimplePaginatorInfo` in the docs as well. If the other types are lacking that information, we should add it.